virttest/utils_conn.py

```diff
@@ -1292,17 +1300,18 @@ def server_setup(self, on_local=False):
             raise ConnMkdirError(self.libvirt_pki_private_dir, output)
 
         if scp_new_cacert == 'no':
-            scp_dict = {servercert_path: self.libvirt_pki_dir,
-                        serverkey_path: self.libvirt_pki_private_dir}
+            params_scp = {'serverkey': serverkey_path,
+                          'servercert': servercert_path}
```

Comment:
Is scp_new_cecert still useful? can we set to scp_list_xxx directly?

Reply:
scp_new_cecert is the indicator to decide what files will be scped. So I think it is still useful.